pomegranate/base.pyx

```diff
@@ -41,16 +35,22 @@ cdef class Model( object ):
 	def set_params(self, state):
 		self.__setstate__(state)
 
-	def copy( self ):
+	def to_json(self):
+		"""Serialize this object into JSON format."""
+
```

Comment:
We have an empty line here, but no empty line in from_json. I'd prefer no lines.